sbi/inference/posterior.py

```diff
@@ -671,6 +671,7 @@ def _match_x_with_theta_batch_shape(x: Tensor, theta: Tensor) -> Tensor:
 
     @property
     def sample_with_mcmc(self) -> bool:
+        """Return 'True' if NeuralPosterior instance is configured to use MCMC in '.sample()'."""
```

Comment:
We prefer `backticks` when quoting code literals.

Reply:
should be fixed with 96555d8